cla_backend/apps/reports/urls.py

```diff
@@ -16,6 +16,7 @@
         name="mi_provider_allocation_extract",
     ),
     url(r"^mi-case-extract/$", views.mi_case_extract, name="mi_case_extract"),
+    url(r"^mi-case-extract/$", views.mi_case_extract_extended, name="mi_case_extract_extended"),
```

Comment:
I believe this url should point to mi-case-extract-extended. 

Currently, it is opening the mi-case-extract page.